cea/default.config

```diff
@@ -408,6 +413,14 @@ solar-window-solstice.type = IntegerParameter
 solar-window-solstice.help = Desired time of solar exposure on the solstice in hours.
 solar-window-solstice.category = Advanced
 
+panel-tilt-angle = 10
+panel-tilt-angle.type = RealParameter
+panel-tilt-angle.help = Solar panel tilt angle if using user-defined panel tilt angle and roof coverage.
+
+max-roof-coverage = 0.8
+max-roof-coverage.type = RealParameter
+max-roof-coverage.help = Maximum roof coverage if using user-defined panel tilt angle and roof coverage.
```

Comment:
please describe how is the calculation done when the radiation threshold is also set

Reply:
- the wording has to echo *type-placement*
- add unit

Reply:
rephrased as requested